packages/app/src/cli/services/payments/extension-to-toml.ts

```diff
@@ -89,6 +94,8 @@ export function buildPaymentsToml<T extends BasePaymentsAppExtensionDeployConfig
   const cliConfig = serialize(dashboardConfig)
   if (cliConfig) delete cliConfig.api_version
 
+  const context = extension.activeVersion?.context || extension.draftVersion?.context || typeToContext(extension.type)
```

Comment:
So if we do have the context in the active version, it means it's a CLI extension, otherwise we fall back to generating the context from `extension.type` for dashboard ones, right?